wof/wofpy_config.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import (absolute_import, division, print_function)
+
+import os
+import shutil
+from docopt import docopt
+
+__all__ = [
```

Comment:
The `wofpy_config` script is quite simple and all it does is copy the user configuration files into a new directory.
We can further simplify the install instructions using this script and change the docs to tell users to use:

`wofpy_config /var/www/timeseries`